modules/ROOT/pages/getting-started.adoc

```diff
@@ -20,6 +20,65 @@ If this is your first time working with the connector, we recommend starting wit
 === Distributed
 
 If a single connector process cannot handle all of your traffic, multiple connector processes can be deployed in "distributed" mode.
+
+.Distributed data flow with ElasticSearch
+[plantuml]
+....
+
+skin rose
+skinparam actorStyle awesome
+
+
+database "Travel\nSample" as travelSample
+
+component "Filter by type\nevent" as vBucketFilter
+
+component "Elastic\nConnector 1" as ecs1
+component "Elastic\nConnector 2" as ecs2
+
+database "ElasticSearch\nAirport\nIndex" as pi1
+database "ElasticSearch\nHotels\nIndex" as pi2
+component "ElasticSearch API" as elasticSearch
+
+travelSample --> vBucketFilter
+vBucketFilter --> ecs1 : airports
+vBucketFilter --> ecs2 : hotels
+
+ecs1 --> pi1
+ecs2 --> pi2
+
+actor "User" as user
+
+pi1 --> elasticSearch
+pi2 --> elasticSearch
+
+user --up-> elasticSearch: query
+
+user <-right- elasticSearch: result
+
+note left of vBucketFilter
+You can use an Eventing function to filter
+the records and dispatch them to
+ separately-configured
```

Comment:
minor nit - but the leading space needs removing (it shows up as a gap in the text in the diagram).